.eslintrc.js

```diff
@@ -149,5 +149,7 @@ module.exports = {
     spyOnProd: true,
     __PROFILE__: true,
     __UMD__: true,
+    TrustedTypes: true,
+    trustedTypes: true,
```

Comment:
Do we really need to support both?

Reply:
In .eslintrc yes. But I think you meant it in a more general way. Currently, the native chrome implementation uses TrustedTypes (with uppercase T) and that is the preferred way to test Trusted Types.

Reply:
It's unfortunate for the bundle size to support both. Can we start with one version, and then switch over to another one when it reaches stable?

Reply:
Yeah, we can do that. Users can set `window.trustedTypes = window.TrustedTypes` before bootstrapping React.